src/style.css

```diff
@@ -0,0 +1,127 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir sans-serif;
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir sans-serif;
+  font-weight: 400;
+  src: url(fonts/Avenir.ttc);
+}
+
+body {
+  /* margin-left: 0; */
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form-field {
+  box-sizing: border-box;
+  margin: 20px 0;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  display: block;
+  width: 100%;
+  padding-left: 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: top 25px left 26px;
+}
+
+.form-field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-field:focus {
+  font-family: Avenir, sans-serif;
+  font-weight: 700;
+  background-image: url(images/Search.svg), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat;
+  background-position: top 25px left 26px;
+  color: #3d4e61;
+}
+
+.text-input::placeholder {
+  color: #3d4e61;
+  opacity: 1;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.text-input {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  outline: none;
+  color: #3d4e61;
+  font-size: 16px;
+}
+
+.form {
+  margin: 20px 0;
+}
+
+/* second bar */
+
+.form-field-small {
+  box-sizing: border-box;
+  height: 42px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  display: block;
+  width: 100%;
+  padding-left: 33px;
+  background-size: 11px;
+  background-repeat: no-repeat;
+  background-position: top 16px left 13px;
+}
+
+.form-field-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.small-image {
+  height: 11px;
+  width: 11px;
+  position: absolute;
+  margin: 15px 0 0 13px;
+  background-image: url(images/Search.svg);
+  background-size: 11px;
+  background-repeat: no-repeat;
+}
+
+button {
+  border: none;
+  background-color: inherit;
+}
+
+.form-field-small:focus {
+  font-family: Avenir, sans-serif;
+  font-weight: 700;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  color: #3d4e61;
+}
+
+.text-input-small {
+  font-family: Avenir, sans-serif;
```

Comment:
No need to set font-family over and over again, just set it in one common class for inputs